packages/profile-sync-controller/src/sdk/env.ts

```diff
@@ -1,10 +1,10 @@
-export const enum Env {
+export enum Env {
```

Comment:
Removing `const enum`'s as they are not compatible with projects that use TS`isolatedModules: true`